zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
we can keep the code here and just add this flag, by default if you are using a checkbox you are allowing multiple, else you are using the list, and in a list, none could be an option. 
` if not checkbox:
      if allow_none:
           choices = choices + [None]
      cont...`
This way we keep a centralized function that handles our inquiry. 

From your example, `pick` can do the same and more than pick_multiple. By default, checkbox based inquiry allows you to pick nothing or multiple while list requires a single choice (i believe), therefore none should be an option if checkbox is none, because we are making a list. 

Am i missing  something?